.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+omit =
```

Comment:
There should be a TODO left here to remove this when the models are incorporated into the rest of the code.